pkg/jx/cmd/test_data/step_create_task/from_yaml/pipeline.yml

```diff
@@ -5,30 +5,40 @@ metadata:
   name: abayer-js-test-repo-really-long-1
   namespace: jx
 spec:
-  params: []
+  params:
```

Comment:
@joseblas, @abayer - ooi, what is your experience in terms of maintenance of these type of tests. It is not quite labour intensive to adjust these tests as changes are made to the code? Just wondering.

Reply:
Indeed, but I think in this case is worth keeping.

Reply:
Ok